kitchen/ph5toms

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-#   Version: 2016.133
-EXEC='ph5tomsAPI'
+#   Version: 2017.167
+EXEC='ph5toms'
```

Comment:
Okay. Keep in mind that the suggested naming convention is all lower case for python modules. 

For best practices see: 
https://python-packaging.readthedocs.io/en/latest/minimal.html#picking-a-name